modules/apps/journal/journal-web/src/main/resources/META-INF/resources/edit_article.jsp

```diff
@@ -356,18 +356,77 @@ journalEditArticleDisplayContext.setViewAttributes();
 							<div id="<portlet:namespace />descriptionMapAsXMLWrapper">
 								<label for="<portlet:namespace />descriptionMapAsXML" id="<portlet:namespace />Aria"><liferay-ui:message key="description" /></label>
 
-								<liferay-ui:input-localized
-									availableLocales="<%= journalEditArticleDisplayContext.getAvailableLocales() %>"
-									defaultLanguageId="<%= journalEditArticleDisplayContext.getDefaultArticleLanguageId() %>"
-									editorName="ckeditor"
-									formName="fm"
-									ignoreRequestValue="<%= journalEditArticleDisplayContext.isChangeStructure() %>"
-									languagesDropdownVisible="<%= false %>"
-									name="descriptionMapAsXML"
-									selectedLanguageId="<%= journalEditArticleDisplayContext.getSelectedLanguageId() %>"
-									type="editor"
-									xml="<%= (article != null) ? article.getDescriptionMapAsXML() : StringPool.BLANK %>"
-								/>
+								<c:choose>
+									<c:when test="<%= true %>">
```

Comment:
Shall this also be gated under LPD-51378 ff?

Reply:
Yup, accidentally committed this